lib/SauceBrowser.js

```diff
@@ -184,7 +184,10 @@ SauceBrowser.prototype.start = function() {
                             }
 
                             self.debug('fetching more results');
-                            setTimeout(wait, 1000);
+
+                            // wait 50ms if results were found
+                            // otherwise wait 500ms (so that we do not flood saucelabs)
+                            setTimeout(wait, res.length && 50 || 500);
```

Comment:
lets be explicit with res.length > 0 and use a ternary here instead of being cute with the expression evaluation :)

If results found why wait at all?

If no results, can we wait longer?